debezium-core/src/main/java/io/debezium/connector/common/BaseSourceTask.java

```diff
@@ -239,7 +239,7 @@ public final void start(Map<String, String> props) {
             setTaskState(DebeziumTaskState.INITIAL);
             config = Configuration.from(props);
 
-            DebeziumOpenLineageEmitter.init(props, connectorName());
+            DebeziumOpenLineageEmitter.init(getMaskedConfigurationMap(props), connectorName());
             DebeziumOpenLineageEmitter.emit(DebeziumOpenLineageEmitter.connectorContext(props, connectorName()), DebeziumTaskState.INITIAL);
 
```

Comment:
You should also add the call to masking for `DebeziumOpenLineageEmitter.connectorContext` method whenever it is used.